Aug 30 21:20:02 ubuntu-s-1vcpu-1gb-sgp1-01 python[303270]: Traceback (most recent call last):
Aug 30 21:20:02 ubuntu-s-1vcpu-1gb-sgp1-01 python[303270]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:02 ubuntu-s-1vcpu-1gb-sgp1-01 python[303270]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:02 ubuntu-s-1vcpu-1gb-sgp1-01 python[303270]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:05 ubuntu-s-1vcpu-1gb-sgp1-01 python[303284]: Traceback (most recent call last):
Aug 30 21:20:05 ubuntu-s-1vcpu-1gb-sgp1-01 python[303284]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:05 ubuntu-s-1vcpu-1gb-sgp1-01 python[303284]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:05 ubuntu-s-1vcpu-1gb-sgp1-01 python[303284]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:08 ubuntu-s-1vcpu-1gb-sgp1-01 python[303291]: Traceback (most recent call last):
Aug 30 21:20:08 ubuntu-s-1vcpu-1gb-sgp1-01 python[303291]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:08 ubuntu-s-1vcpu-1gb-sgp1-01 python[303291]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:08 ubuntu-s-1vcpu-1gb-sgp1-01 python[303291]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:10 ubuntu-s-1vcpu-1gb-sgp1-01 python[303294]: Traceback (most recent call last):
Aug 30 21:20:10 ubuntu-s-1vcpu-1gb-sgp1-01 python[303294]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:10 ubuntu-s-1vcpu-1gb-sgp1-01 python[303294]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:10 ubuntu-s-1vcpu-1gb-sgp1-01 python[303294]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:13 ubuntu-s-1vcpu-1gb-sgp1-01 python[303295]: Traceback (most recent call last):
Aug 30 21:20:13 ubuntu-s-1vcpu-1gb-sgp1-01 python[303295]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:13 ubuntu-s-1vcpu-1gb-sgp1-01 python[303295]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:13 ubuntu-s-1vcpu-1gb-sgp1-01 python[303295]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:15 ubuntu-s-1vcpu-1gb-sgp1-01 python[303297]: Traceback (most recent call last):
Aug 30 21:20:15 ubuntu-s-1vcpu-1gb-sgp1-01 python[303297]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:15 ubuntu-s-1vcpu-1gb-sgp1-01 python[303297]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:15 ubuntu-s-1vcpu-1gb-sgp1-01 python[303297]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:18 ubuntu-s-1vcpu-1gb-sgp1-01 python[303301]: Traceback (most recent call last):
Aug 30 21:20:18 ubuntu-s-1vcpu-1gb-sgp1-01 python[303301]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:18 ubuntu-s-1vcpu-1gb-sgp1-01 python[303301]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:18 ubuntu-s-1vcpu-1gb-sgp1-01 python[303301]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303302]: Traceback (most recent call last):
Aug 30 21:20:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303302]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303302]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303302]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:23 ubuntu-s-1vcpu-1gb-sgp1-01 python[303304]: Traceback (most recent call last):
Aug 30 21:20:23 ubuntu-s-1vcpu-1gb-sgp1-01 python[303304]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:23 ubuntu-s-1vcpu-1gb-sgp1-01 python[303304]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:23 ubuntu-s-1vcpu-1gb-sgp1-01 python[303304]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303308]: Traceback (most recent call last):
Aug 30 21:20:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303308]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303308]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303308]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303309]: Traceback (most recent call last):
Aug 30 21:20:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303309]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303309]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303309]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:31 ubuntu-s-1vcpu-1gb-sgp1-01 python[303312]: Traceback (most recent call last):
Aug 30 21:20:31 ubuntu-s-1vcpu-1gb-sgp1-01 python[303312]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:31 ubuntu-s-1vcpu-1gb-sgp1-01 python[303312]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:31 ubuntu-s-1vcpu-1gb-sgp1-01 python[303312]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303315]: Traceback (most recent call last):
Aug 30 21:20:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303315]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303315]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303315]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[303318]: Traceback (most recent call last):
Aug 30 21:20:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[303318]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[303318]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[303318]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:39 ubuntu-s-1vcpu-1gb-sgp1-01 python[303320]: Traceback (most recent call last):
Aug 30 21:20:39 ubuntu-s-1vcpu-1gb-sgp1-01 python[303320]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:39 ubuntu-s-1vcpu-1gb-sgp1-01 python[303320]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:39 ubuntu-s-1vcpu-1gb-sgp1-01 python[303320]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303324]: Traceback (most recent call last):
Aug 30 21:20:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303324]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303324]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303324]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[303326]: Traceback (most recent call last):
Aug 30 21:20:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[303326]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[303326]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[303326]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303327]: Traceback (most recent call last):
Aug 30 21:20:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303327]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303327]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303327]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[303331]: Traceback (most recent call last):
Aug 30 21:20:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[303331]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[303331]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[303331]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303340]: Traceback (most recent call last):
Aug 30 21:20:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303340]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303340]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303340]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303541]: Traceback (most recent call last):
Aug 30 21:20:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303541]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303541]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303541]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:20:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303545]: Traceback (most recent call last):
Aug 30 21:20:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303545]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:20:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303545]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:20:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303545]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303548]: Traceback (most recent call last):
Aug 30 21:21:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303548]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303548]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303548]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303552]: Traceback (most recent call last):
Aug 30 21:21:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303552]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303552]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303552]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:07 ubuntu-s-1vcpu-1gb-sgp1-01 python[303556]: Traceback (most recent call last):
Aug 30 21:21:07 ubuntu-s-1vcpu-1gb-sgp1-01 python[303556]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:07 ubuntu-s-1vcpu-1gb-sgp1-01 python[303556]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:07 ubuntu-s-1vcpu-1gb-sgp1-01 python[303556]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303558]: Traceback (most recent call last):
Aug 30 21:21:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303558]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303558]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303558]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[303561]: Traceback (most recent call last):
Aug 30 21:21:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[303561]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[303561]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[303561]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303563]: Traceback (most recent call last):
Aug 30 21:21:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303563]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303563]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303563]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303566]: Traceback (most recent call last):
Aug 30 21:21:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303566]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303566]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303566]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303569]: Traceback (most recent call last):
Aug 30 21:21:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303569]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303569]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[303569]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303572]: Traceback (most recent call last):
Aug 30 21:21:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303572]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303572]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303572]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303574]: Traceback (most recent call last):
Aug 30 21:21:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303574]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303574]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303574]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303576]: Traceback (most recent call last):
Aug 30 21:21:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303576]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303576]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[303576]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303579]: Traceback (most recent call last):
Aug 30 21:21:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303579]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303579]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303579]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303581]: Traceback (most recent call last):
Aug 30 21:21:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303581]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303581]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303581]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303585]: Traceback (most recent call last):
Aug 30 21:21:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303585]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303585]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303585]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303586]: Traceback (most recent call last):
Aug 30 21:21:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303586]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303586]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303586]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303589]: Traceback (most recent call last):
Aug 30 21:21:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303589]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303589]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303589]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303592]: Traceback (most recent call last):
Aug 30 21:21:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303592]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303592]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303592]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303595]: Traceback (most recent call last):
Aug 30 21:21:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303595]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303595]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303595]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303597]: Traceback (most recent call last):
Aug 30 21:21:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303597]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303597]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303597]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303599]: Traceback (most recent call last):
Aug 30 21:21:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303599]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303599]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303599]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303602]: Traceback (most recent call last):
Aug 30 21:21:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303602]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303602]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303602]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303604]: Traceback (most recent call last):
Aug 30 21:21:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303604]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303604]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303604]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:21:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303606]: Traceback (most recent call last):
Aug 30 21:21:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303606]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:21:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303606]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:21:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303606]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303609]: Traceback (most recent call last):
Aug 30 21:22:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303609]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303609]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303609]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303611]: Traceback (most recent call last):
Aug 30 21:22:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303611]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303611]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303611]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303613]: Traceback (most recent call last):
Aug 30 21:22:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303613]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303613]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303613]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303617]: Traceback (most recent call last):
Aug 30 21:22:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303617]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303617]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303617]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303622]: Traceback (most recent call last):
Aug 30 21:22:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303622]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303622]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303622]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303624]: Traceback (most recent call last):
Aug 30 21:22:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303624]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303624]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303624]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303627]: Traceback (most recent call last):
Aug 30 21:22:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303627]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303627]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[303627]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303629]: Traceback (most recent call last):
Aug 30 21:22:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303629]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303629]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303629]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303631]: Traceback (most recent call last):
Aug 30 21:22:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303631]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303631]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303631]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303636]: Traceback (most recent call last):
Aug 30 21:22:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303636]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303636]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[303636]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303640]: Traceback (most recent call last):
Aug 30 21:22:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303640]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303640]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303640]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303642]: Traceback (most recent call last):
Aug 30 21:22:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303642]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303642]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303642]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303645]: Traceback (most recent call last):
Aug 30 21:22:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303645]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303645]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[303645]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303649]: Traceback (most recent call last):
Aug 30 21:22:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303649]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303649]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303649]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303651]: Traceback (most recent call last):
Aug 30 21:22:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303651]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303651]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303651]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303653]: Traceback (most recent call last):
Aug 30 21:22:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303653]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303653]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[303653]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303656]: Traceback (most recent call last):
Aug 30 21:22:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303656]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303656]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303656]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303658]: Traceback (most recent call last):
Aug 30 21:22:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303658]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303658]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303658]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303660]: Traceback (most recent call last):
Aug 30 21:22:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303660]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303660]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303660]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303663]: Traceback (most recent call last):
Aug 30 21:22:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303663]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303663]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303663]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303665]: Traceback (most recent call last):
Aug 30 21:22:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303665]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303665]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303665]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303668]: Traceback (most recent call last):
Aug 30 21:22:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303668]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303668]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303668]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:22:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303669]: Traceback (most recent call last):
Aug 30 21:22:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303669]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:22:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303669]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:22:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303669]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303674]: Traceback (most recent call last):
Aug 30 21:23:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303674]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303674]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303674]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303676]: Traceback (most recent call last):
Aug 30 21:23:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303676]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303676]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303676]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303677]: Traceback (most recent call last):
Aug 30 21:23:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303677]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303677]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303677]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303681]: Traceback (most recent call last):
Aug 30 21:23:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303681]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303681]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303681]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303683]: Traceback (most recent call last):
Aug 30 21:23:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303683]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303683]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303683]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303684]: Traceback (most recent call last):
Aug 30 21:23:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303684]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303684]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303684]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303687]: Traceback (most recent call last):
Aug 30 21:23:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303687]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303687]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303687]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303691]: Traceback (most recent call last):
Aug 30 21:23:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303691]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303691]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303691]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:21 ubuntu-s-1vcpu-1gb-sgp1-01 python[303693]: Traceback (most recent call last):
Aug 30 21:23:21 ubuntu-s-1vcpu-1gb-sgp1-01 python[303693]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:21 ubuntu-s-1vcpu-1gb-sgp1-01 python[303693]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:21 ubuntu-s-1vcpu-1gb-sgp1-01 python[303693]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303697]: Traceback (most recent call last):
Aug 30 21:23:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303697]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303697]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303697]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303701]: Traceback (most recent call last):
Aug 30 21:23:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303701]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303701]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303701]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:29 ubuntu-s-1vcpu-1gb-sgp1-01 python[303702]: Traceback (most recent call last):
Aug 30 21:23:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303702]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303702]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303702]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303704]: Traceback (most recent call last):
Aug 30 21:23:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303704]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303704]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303704]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303708]: Traceback (most recent call last):
Aug 30 21:23:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303708]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303708]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303708]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303709]: Traceback (most recent call last):
Aug 30 21:23:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303709]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303709]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303709]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303711]: Traceback (most recent call last):
Aug 30 21:23:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303711]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303711]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303711]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303715]: Traceback (most recent call last):
Aug 30 21:23:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303715]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303715]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303715]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303718]: Traceback (most recent call last):
Aug 30 21:23:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303718]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303718]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[303718]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303722]: Traceback (most recent call last):
Aug 30 21:23:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303722]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303722]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303722]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303725]: Traceback (most recent call last):
Aug 30 21:23:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303725]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303725]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303725]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303728]: Traceback (most recent call last):
Aug 30 21:23:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303728]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303728]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303728]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303729]: Traceback (most recent call last):
Aug 30 21:23:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303729]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303729]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303729]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:23:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[303731]: Traceback (most recent call last):
Aug 30 21:23:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[303731]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:23:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[303731]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:23:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[303731]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303735]: Traceback (most recent call last):
Aug 30 21:24:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303735]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303735]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[303735]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303736]: Traceback (most recent call last):
Aug 30 21:24:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303736]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303736]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[303736]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303738]: Traceback (most recent call last):
Aug 30 21:24:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303738]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303738]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[303738]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303744]: Traceback (most recent call last):
Aug 30 21:24:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303744]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303744]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[303744]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303745]: Traceback (most recent call last):
Aug 30 21:24:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303745]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303745]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[303745]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303747]: Traceback (most recent call last):
Aug 30 21:24:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303747]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303747]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[303747]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303751]: Traceback (most recent call last):
Aug 30 21:24:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303751]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303751]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:16 ubuntu-s-1vcpu-1gb-sgp1-01 python[303751]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303752]: Traceback (most recent call last):
Aug 30 21:24:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303752]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303752]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[303752]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303755]: Traceback (most recent call last):
Aug 30 21:24:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303755]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303755]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[303755]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303759]: Traceback (most recent call last):
Aug 30 21:24:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303759]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303759]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[303759]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303761]: Traceback (most recent call last):
Aug 30 21:24:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303761]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303761]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[303761]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303763]: Traceback (most recent call last):
Aug 30 21:24:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303763]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303763]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[303763]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303766]: Traceback (most recent call last):
Aug 30 21:24:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303766]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303766]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[303766]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303769]: Traceback (most recent call last):
Aug 30 21:24:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303769]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303769]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:35 ubuntu-s-1vcpu-1gb-sgp1-01 python[303769]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303770]: Traceback (most recent call last):
Aug 30 21:24:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303770]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303770]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[303770]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303772]: Traceback (most recent call last):
Aug 30 21:24:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303772]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303772]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[303772]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303776]: Traceback (most recent call last):
Aug 30 21:24:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303776]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303776]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:43 ubuntu-s-1vcpu-1gb-sgp1-01 python[303776]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303777]: Traceback (most recent call last):
Aug 30 21:24:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303777]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303777]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[303777]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303780]: Traceback (most recent call last):
Aug 30 21:24:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303780]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303780]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:48 ubuntu-s-1vcpu-1gb-sgp1-01 python[303780]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303785]: Traceback (most recent call last):
Aug 30 21:24:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303785]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303785]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[303785]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303786]: Traceback (most recent call last):
Aug 30 21:24:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303786]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303786]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:53 ubuntu-s-1vcpu-1gb-sgp1-01 python[303786]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303788]: Traceback (most recent call last):
Aug 30 21:24:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303788]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303788]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[303788]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
Aug 30 21:24:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303792]: Traceback (most recent call last):
Aug 30 21:24:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303792]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 14, in <module>
Aug 30 21:24:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303792]:     raise RuntimeError("❌ Telegram bot token не найден в окружении (.env)")
Aug 30 21:24:58 ubuntu-s-1vcpu-1gb-sgp1-01 python[303792]: RuntimeError: ❌ Telegram bot token не найден в окружении (.env)
